app/main/subsystems/wallets.ts

```diff
@@ -12,7 +12,14 @@ import { Lifecycle } from "app/main/lifecycle"
 export class WalletStorage {
 
   /** The JsonAesLevelStorage */
-  private storage: Storage<Buffer, JsonSerializable, Buffer, Buffer> | undefined = undefined
+  private _storage: Storage<Buffer, JsonSerializable, Buffer, Buffer> | undefined = undefined
```

Comment:
I'm unsure about usage of underscores for private class properties, as we have started using underscores to tell the compiler not to verify unused variables.

Maybe we could name them as `whateverReference` (`storageReference` in this case) if they are references and `whateverValue` if they are values.